### PR TITLE
Adapt Framework Issue/#1955

### PR DESCRIPTION
**Pull request by jamesrea83**
re https://github.com/adaptlearning/adapt_framework/issues/1955

### CODE REVIEW

**Comment by OscarArango** [comment]
Not sure if this is related, but apparently now when using the shortcodes, when viewing the course, it no longer pulls the data, instead it shows the {{{score}}} as it's written with the rest of the text.  This is on:

Adapt Authoring 0.4.1
framework 2.2.5

thanks,

**Comment by moloko** [comment]
@OscarArango just to be clear, are you saying that happens when you apply the changes in this pull request?

**Comment by moloko** [comment]
@jamesrea83 there are [some comments from the original PR](https://github.com/adaptlearning/adapt-contrib-assessmentResults/pull/37) that you need to take into account here.

**Comment by OscarArango** [comment]
@moloko i'm not sure it has anything to do with this build.  Unfortunately GitHub has no way to list bugs or issues that I can see.  I end up posting under pull requests so that the issue doesn't go unnoticed.  I shouldn't post here, but it's like I have to let the developer know.  If I should post somewhere other then here please let me know.  Adapt forum was not a good place since most posts get buried.

**Comment by moloko** [comment]
@OscarArango if you look in bower.json for any plugin there should be a link for where to log issues. For the framework or any of the plugins supported by the core team it'll always be https://github.com/adaptlearning/adapt_framework/issues/new

with regards to this particular problem you might want to have a look at [this post](https://community.adaptlearning.org/mod/forum/discuss.php?d=2234) before logging an issue

**Comment by OscarArango** [comment]
Thanks Matt,

 

Your response was super helpful, I’ll be sure to use the proper channel next time.  As for the issue I was having, your answer is correct, old theme overwriting the new theme files.  Thanks a million for this response.

 

Thanks,

File: properties.schema [hunk 1]
```diff
@@ -29,6 +29,19 @@
       "validators": [],
       "help": "Controls whether this component will be visible or hidden prior to the assessment having been completed."
     },
+    "_setCompletionOn": {
+      "type": "string",
+      "required": false,
+      "default": "inview",
+      "title": "Criteria for setting completion on this component.",
```
Comment (moloko) [inline properties.schema]: Could you set the title to 'Set completion on:'?

File: properties.schema [hunk 2]
```diff
@@ -29,6 +29,19 @@
...
+      "enum": ["inview", "pass"],
+      "inputType": {
+        "type": "Select",
+        "options": ["inview", "pass"]
+      },
+      "validators": [],
+      "help": "Can set to complete when the component is inview, or when the assessment is passed."
```
Comment (moloko) [inline properties.schema]: Please change to "Set to 'pass' to have the component mark as completed only if the learner passed the associated assessment."

File: js/adapt-contrib-assessmentResults.js [hunk 3]
```diff
@@ -153,6 +153,16 @@ define([
             }
         },
 
+        checkCompletion: function() {
+            var setCompletionOn = this.model.get('_setCompletionOn');
```
Comment (moloko) [inline js/adapt-contrib-assessmentResults.js]: I think you should be able to reduce this function down to:
```js
if (this.model.get('_setCompletionOn') === "pass" && !this.model.get('isPass')) {
    return false;
}

return true;
```

**Comment by jamesrea83** [comment]
Apologies, this one is getting complicated. Moving to new PR.

**Pull request closed without merging**